scripts/power.py

```diff
@@ -27,41 +31,53 @@ def init(self):
         )
         self.parser_reboot2dfu.set_defaults(func=self.reboot2dfu)
 
-    def _get_flipper(self):
-        if not (port := resolve_port(self.logger, self.args.port)):
+    def _get_flipper(self, retry_count: Optional[int] = 1):
+        port = None
+        self.logger.info(f"Attempting to find flipper with {retry_count} attempts.")
+
+        for i in range(retry_count):
+            time.sleep(1)
+            self.logger.info(f"Attempting to find flipper #{i}.")
+
+            if port := resolve_port(self.logger, self.args.port):
+                self.logger.info(f"Found flipper at {port}")
+                break
+
+        if not port:
+            self.logger.info(f"Failed to find flipper")
             return None
 
         flipper = FlipperStorage(port)
         flipper.start()
         return flipper
 
     def power_off(self):
-        if not (flipper := self._get_flipper()):
-            return 1
+        if not (flipper := self._get_flipper(retry_count=10)):
+            sys.exit(1)
```

Comment:
Why? Return is doing exactly the same but gives upper level ability to process it